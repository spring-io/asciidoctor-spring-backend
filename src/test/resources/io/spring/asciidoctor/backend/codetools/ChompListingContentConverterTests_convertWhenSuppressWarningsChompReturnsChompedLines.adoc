= Test Document
:chomp: suppresswarnings

[source,java]
----
public class Example {

	@SuppressWarnings("unused")
	private String field;

}
----
