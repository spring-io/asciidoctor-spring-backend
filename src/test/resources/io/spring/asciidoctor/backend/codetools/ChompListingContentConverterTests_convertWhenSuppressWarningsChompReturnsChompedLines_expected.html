public class Example {

	private String field;

}